src/components/timeline/TimelineLockControl.svelte

```diff
@@ -0,0 +1,67 @@
+<script lang="ts">
+  import LockIcon from '@nasa-jpl/stellar/icons/svg/lock.svg?component';
+  import UnlockIcon from '@nasa-jpl/stellar/icons/svg/unlock.svg?component';
+  import { onMount } from 'svelte';
+  import { timelineLockStatus } from '../../stores/views';
+  import { TimelineLockStatus } from '../../utilities/timeline';
+  import { tooltip } from '../../utilities/tooltip';
+
+  let onKeydown = e => {
```

Comment:
Is there a specific reason you made these variables to anon functions and not named functions?

Reply:
No specific reason, just default by habit. Happy to switch up if named functions are more idiomatic.

Reply:
Yeah let's use the named functions just for idiomatic consistency.
I'll make a note to add this to our non-existent (but soon to be) style guide.